examples/nlp/md/tweet-classification-using-tfdf.md

```diff
@@ -347,26 +347,4582 @@ model_2.fit(train_ds)
+
+```
+</div>
+The model.summary() method prints a variety of information about your decision tree model, including model type, task, input features, and feature importance.
+
+
+```python
+print("model_1 summary: ")
+print(model_1.summary())
+print()
+print("model_2 summary: ")
+print(model_2.summary())
+```
+
+<div class="k-default-codeblock">
+```
+model_1 summary: 
+Model: "gradient_boosted_trees_model"
+_________________________________________________________________
+ Layer (type)                Output Shape              Param #   
+=================================================================
+ model (Functional)          (None, 512)               256797824 
+                                                                 
+=================================================================
+Total params: 256,797,825
+Trainable params: 0
+Non-trainable params: 256,797,825
+_________________________________________________________________
+Type: "GRADIENT_BOOSTED_TREES"
+Task: CLASSIFICATION
+Label: "__LABEL"
+```
+</div>
+    
+<div class="k-default-codeblock">
+```
+Input Features (512):
+	model/keras_layer/StatefulPartitionedCall:0.0
```

Comment:
This summary is pure noise, no added falue for the reader. We should not print it.

Reply:
Does this have anything to do with me? Or are you talking to someone else?

Reply:
This is content that's being added to the `md` file as part of your PR, so it has to do with you. I suggest editing the Python file to remove the corresponding statements.